lib/src/test/java/edu/stanford/futuredata/macrobase/analysis/summary/apriori/APrioriSummarizerTest.java

```diff
@@ -1,155 +0,0 @@
-package edu.stanford.futuredata.macrobase.analysis.summary.apriori;
-
-import edu.stanford.futuredata.macrobase.analysis.classify.ArithmeticClassifier;
-import edu.stanford.futuredata.macrobase.analysis.classify.PercentileClassifier;
-import edu.stanford.futuredata.macrobase.datamodel.DataFrame;
-import edu.stanford.futuredata.macrobase.datamodel.Schema;
-import edu.stanford.futuredata.macrobase.ingest.CSVDataFrameParser;
-import edu.stanford.futuredata.macrobase.ingest.DataFrameLoader;
-import org.junit.Test;
-
-import java.util.*;
-
-import static junit.framework.TestCase.assertEquals;
-import static org.junit.Assert.assertTrue;
-
-public class APrioriSummarizerTest {
-    @Test
-    public void testSimple() throws Exception {
```

Comment:
@kraftp We need to keep this test around—we still need tests of the BasicBatchPipeline.

Reply:
Done